symengine/ruby/valgrind_test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+valgrind --tool=memcheck --num-callers=15 --partial-loads-ok=yes --undef-value-errors=no ruby -S rspec spec
```

Comment:
Oh that worked! So for making this work, I need to be in the root dir of the ruby wrappers. Since, this script will only be used in the ruby wrappers, shall I move this to the `bin` dir in the root dir of the ruby wrappers rather than the root dir of the symengine project?